internal/ovirt/resource_ovirt_vm_test.go

```diff
@@ -187,6 +187,53 @@ resource "ovirt_vm" "foo" {
 		},
 	)
 }
+
+func TestVMResourceVMType(t *testing.T) {
+	t.Parallel()
+
+	p := newProvider(newTestLogger(t))
+	clusterID := p.getTestHelper().GetClusterID()
+	templateID := p.getTestHelper().GetBlankTemplateID()
+	config := fmt.Sprintf(
+		`
+provider "ovirt" {
+	mock = true
+}
+
+resource "ovirt_vm" "foo" {
+	cluster_id  = "%s"
+	template_id = "%s"
+    name        = "test"
+    vm_type     = "server"
+}
+`,
+		clusterID,
+		templateID,
+	)
+
+	resource.UnitTest(
+		t, resource.TestCase{
+			ProviderFactories: p.getProviderFactories(),
+			Steps: []resource.TestStep{
+				{
+					Config: config,
+					Check: resource.ComposeTestCheckFunc(
+						resource.TestMatchResourceAttr(
```

Comment:
Can we check the created VM instead of the return value?

Reply:
You mean via the test helper? Sure thing.